src/library_syscall.js

```diff
@@ -685,6 +699,7 @@ var SyscallsLibrary = {
     return pos;
   },
   __syscall_fcntl64__deps: ['$setErrNo'],
+  __syscall_fcntl64__sig: 'iipp',
```

Comment:
```suggestion
  __syscall_fcntl64__sig: 'iiip',
```
`cmd` is not a pointer.